Fix alignment of tip button on Twitter.

mkarolin opened the pull request:
Fixes https://github.com/brave/brave-browser/issues/26880
Fixes https://github.com/brave/brave-browser/issues/27256

zenparsing commented on scripts/brave_rewards/publisher/twitter/tipping.ts:
```diff
@@ -279,14 +266,17 @@ export const configure = () => {
-          actions = replyButton.parentElement?.parentElement
-        }
+      const groups = tweets[i].querySelectorAll('[role="group"]')
+      for (let j = 0; j < groups.length; ++j) {
```
Looks like this could use for-of:

```js
for (let group of groups) {
  ...
}
```

mkarolin replied:
Thanks! Updated.

emerick reviewed (approved):
LGTM

mkarolin commented:
Updated one more time. As it turns out `aria-label`'s value is localized so the previous solution would only work in English and then would just fallback on the last `group`. Changed to look for a `group` of more than 3 `button`s.
